http-gateways/SUBDOMAIN_GATEWAY.md

```diff
@@ -0,0 +1,164 @@
+# Subdomain Gateway Specification
+
+![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Authors**:
+
+- Marcin Rataj ([@lidel](https://github.com/lidel))
+
+----
+
+**Abstract**
+
+Subdomain Gateway is an extension of  [PATH_GATEWAY.md](./PATH_GATEWAY.md) that
+enables website hosting compatible with web browsers relative pathing and
+security model of the web. Below should be read as a delta on top of that spec.
+
+Summary:
+
+- data is requested by CID placed in `Host` header
```

Comment:
Should we mention v1? we mention it in Host header details, but maybe a bit hidden

Reply:
Good idea, added in https://github.com/ipfs/specs/pull/283/commits/a414411c44d552e6a2d35266029083e3fd479adf